### DISCUSSION TITLE
Unable to update relationship fields

### DISCUSSION THREAD

**Discussion by hshifaz** [Budibase/budibase#1705] [Help] [answered]
I'm unable to update relationship records from frontend (UI Screen).

It shows "Row Saved", but it's not. Please find attached screenshots
![relationship update issue 1](https://user-images.githubusercontent.com/14042347/121763671-c7ec9880-caf2-11eb-8cfb-760b2c68fdd3.PNG)

I tried to change to a different options and this is the result.

![relationship update issue 2](https://user-images.githubusercontent.com/14042347/121763675-ccb14c80-caf2-11eb-801b-a412767bf19b.PNG)

When I go back to the edit screen the field shows "Choose an option".

Below is how I've Defined the Save button Actions. All other fields works fine. But the relationship field doesn't update.

![relationship update issue 3](https://user-images.githubusercontent.com/14042347/121763715-15690580-caf3-11eb-9de7-88a9b52542aa.PNG)

**Comment by mjashanks** [comment, marked as answer]
Hi there. I'm assuming that you are using an internal table here.

You shouldn't need to declare the fields in your save record action. This should only be used if you want to send some custom value, that is not tied to a field by the same name. Form example, you might want to always set the status `Pending` for new records.

So, your Save Record action should work like this:
![image](https://user-images.githubusercontent.com/3524181/121765836-a072eb80-cb45-11eb-91f6-36e57d04a886.png)

All the same, I think you have found a bug. What you are doing should also work!

**Reply by hshifaz** [reply]
Thank you @mjashanks that works.

If only the documentation had this part explained.
[https://docs.budibase.com/design/actions](https://docs.budibase.com/design/actions) is empty.

**Reply by mjashanks** [reply]
Yes, thanks - we have work to do on docs 🙂
